src/utils/fs.ts

```diff
@@ -278,8 +278,18 @@ export function removeDependency (options: DefinitionOptions) {
 
   // Remove the dependency from typings.
   function remove (path: string, file: string, dtsFile: string) {
-    return promiseFinally(rimraf(path), () => {
-      return transformDtsFile(dtsFile, typings => typings.filter(x => x !== file))
+    const libName: string = path.split('/').slice(-1)[0]
```

Comment:
We can't just split on `/` because Windows uses a different separator. It'd probably be easier to just get the options passed through.